### DISCUSSION TITLE
Hijri Date

### DISCUSSION THREAD

**Discussion by c7dl8p** [DanielRendox/RoutineTracker#21] [Feature Requests]
plz add an option to change the date to hijri Date
![Screenshot_20240606-183608](https://github.com/DanielRendox/RoutineTracker/assets/112339868/d0789bb4-aeb5-4c52-a0df-ceca094b1c17)

**Comment by DanielRendox** [comment]
Hmm, I didn't think about Alternative calendars. Thank you for the suggestion, we may consider adding this in the long term.
